product_pricelist_show_product_ref/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Show Product Ref on Product Pricelist",
+    "summary": "Show Product Ref on Product Pricelist",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Product",
+    "website": "https://www.github.com/OCA/product-attribute",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["sale_management"],
```

Comment:
```suggestion
    "depends": ["sale_management"],
    "maintainers": ["joao-p-marques"],
```
Maybe add a maintainer in manifest it's a good idea in future.

Reply:
Ok, thanks for the tip! Already made the changes :+1: